server/lessons/en/analyze-live-twitter.html

```diff
@@ -145,10 +145,10 @@ <h4>Next</h4>
     <h4>Continue learning:</h4>
     <ol>
       <li>
-        <a href="/lessons/custom-alerts/">Monitor data and set custom alerts</a>
+        <a href="custom-alerts">Monitor data and set custom alerts</a>
       </li>
       <li>
-        <a href="/lessons/clean-and-standardize/">Scrape a webpage to extract or monitor content</a>
+        <a href="clean-and-standardize">Scrape a webpage to extract or monitor content</a>
```

Comment:
The text should be "Clean and standardize data"

Reply:
Actually, I think it makes more sense to link to `scrape-using-xpath`